### PR TITLE
infoucarsinan/issue3

### PR DESCRIPTION
**Pull request by infoucarsinan**
Register page functionalty added.

### CODE REVIEW

File: src/app/app.tsx [hunk 1]
```diff
@@ -1,55 +1,29 @@
...
-              Vite Docs
-            </a>
-          </p>
-          <Switch>
-            <Route path="/about">
-              <main>About</main>
-            </Route>
-            <Route path="/">
-              <main>Home</main>
+    <div>
```
Comment (phlgr) [inline src/app/app.tsx:16]: This div can be safely removed

File: src/app/pages/Register/Register.module.css [hunk 2]
```diff
@@ -0,0 +1,23 @@
+.registrationForm {
+  /* display: grid; */
```
Comment (phlgr) [inline src/app/pages/Register/Register.module.css:2]:
```suggestion
```

**Review by phlgr — COMMENTED** [review]
This PR is quite big and therefore not very pleasant to review. Please split the different party like frontend, backend and db connection into different PRs next time.

File: src/app/pages/Register/Register.module.css (hunk 2 above)
Comment (infoucarsinan) [reply src/app/pages/Register/Register.module.css:2]: ok

File: src/app/app.tsx (hunk 1 above)
Comment (infoucarsinan) [reply src/app/app.tsx:16]: ok

**Pull request merged**